pages/windows/get-date.md

```diff
@@ -0,0 +1,21 @@
+# Get-Date
+
+> Gets the current date and time.
+> This command can only be used through PowerShell.
+> More information: <https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-date>.
+
+- Get the current date and time:
+
+`Get-Date`
+
+- Get the date and time with a .NET format specifier:
+
+`Get-Date -Format {{"dddd MM/dd/yyyy HH:mm K"}}`
```

Comment:
The quotes should be outside the token syntax, also I think the ISO format is a better international example
```suggestion
`Get-Date -Format "{{dddd MM/dd/yyyy HH:mm K}}"`
```